### PR TITLE
Update link

### PR DESCRIPTION
**Pull request by CalMacCQ**
I have updated the link to the quantinuum web interface to the correct one and also added the quantinuum logo.

### CODE REVIEW

File: .github/workflows/docs/intro.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 pytket-extensions
 =================
 
-.. image:: CQCLogo.png
-   :width: 120px
+.. image:: Quantinuum_logo.png
+   :width: 200px
```
Comment (cqc-melf) [inline .github/workflows/docs/intro.txt:5]: Have you checked if the bigger image is displayed in the right way?

File: docs/intro.txt [hunk 2]
```diff
@@ -1,8 +1,8 @@
 pytket-quantinuum
 ==================================
 
-.. image:: CQCLogo.png
-   :width: 120px
+.. image:: Quantinuum_logo.png
```
Comment (cqc-melf) [inline docs/intro.txt:4]: See above

File: .github/workflows/docs/intro.txt [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
    :align: right
```
Comment (cqc-melf) [inline .github/workflows/docs/intro.txt:8]: Do you want to replace the `.. _CQC: https://cambridgequantum.com` link, too?

**Review by cqc-melf — CHANGES_REQUESTED** [review]
Have you added the Quantinuum_logo.png file somewhere? I have not seen that?

File: .github/workflows/docs/intro.txt (hunk 3 above)
Comment (CalMacCQ) [reply .github/workflows/docs/intro.txt:8]: Thanks for spotting that - I've replaced it.

File: .github/workflows/docs/intro.txt (hunk 1 above)
Comment (CalMacCQ) [reply .github/workflows/docs/intro.txt:5]: Yes I have built the docs locally and it seems like an okay size. Do you think it looks okay?

Comment (cqc-melf) [reply .github/workflows/docs/intro.txt:5]: I have not taken a look yet, will build it now, too.

File: .github/workflows/docs/intro.txt (hunk 3 above)
Comment (cqc-melf) [reply .github/workflows/docs/intro.txt:8]: Perfect.

**Comment by cqc-melf** [comment]
The run of the checks stopped in the middle, I have started a rerun to find out if it works no.

File: .github/workflows/docs/intro.txt (hunk 1 above)
Comment (cqc-melf) [reply .github/workflows/docs/intro.txt:5]: Are we allowed within the CI to make the "Q" a bit bigger and keep the "quantinuum" in the same size?

**Review by cqc-melf — APPROVED** [review]
From my side this can be merged now. :)

**Pull request merged**